evap/staff/views.py

```diff
@@ -1182,6 +1186,7 @@ def notify_reward_points(users, **kwargs):
 
     if form.is_valid():
         form.save()
+        delete_navbar_cache_for_user(user)
```

Comment:
i think i would just `delete_navbar_cache_for_users([user]). it's a tiny bit silly but so is having an extra method for that. your call.

Reply:
actually already changed that :)